test/OpenTelemetry.Resources.AssemblyMetadata.Tests/OpenTelemetry.Resources.AssemblyMetadata.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(RepoRoot)\src\OpenTelemetry.Resources.AssemblyMetadata\_.targets" />
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
```

Comment:
This comment is redundant if .NET Framework isn't included in the test matrix.

```suggestion
```

Is there a reason why this isn't including .NET Framework like below?

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/da1620d0b3a2d0de4c056bde47f0bb3138a128e8/test/OpenTelemetry.Resources.Host.Tests/OpenTelemetry.Resources.Host.Tests.csproj#L4-L6

Reply:
Ope, no reason at all. I copy/pasta'd one of the other test projects to use as a template, so either it didn't include that, or (more likely) I dropped it by mistake.

Bringing this back now...